gradio/interface.py

```diff
@@ -264,8 +264,8 @@ def __init__(
         ]:
             for o in self.output_components:
                 assert isinstance(o, IOComponent)
-                o.interactive = False  # Force output components to be non-interactive
-
+                if o.interactive is None:
+                    o.interactive = False
```

Comment:
I'd say the comment was valuable.
```suggestion
                if o.interactive is None:
                    # Unless explicitly otherwise specified, 
                    # force output components to be non-interactive
                    o.interactive = False
```